rq/queue.py

```diff
@@ -699,10 +714,13 @@ def lpop(cls, queue_keys, timeout, connection: t.Optional['Redis'] = None):
         if timeout is not None:  # blocking variant
             if timeout == 0:
                 raise ValueError('RQ does not support indefinite timeouts. Please pick a timeout value > 0')
+            logger.debug(f"Starting BLPOP operation for queues {green(queue_keys)} with timeout of {timeout}")
             result = connection.blpop(queue_keys, timeout)
             if result is None:
+                logger.debug(f"BLPOP Timeout, no jobs found on queues {green(queue_keys)}")
                 raise DequeueTimeout(timeout, queue_keys)
             queue_key, job_id = result
+            logger.debug(f"Found job {blue(job_id)} on queue {green(queue_key)}")
```

Comment:
One more minor nitpick before I merge this in. Can we slightly rephrase this to `Dequeued job {blue(job_id)} from queue {green(queue_key)}`? This should make it clear that the job has been popped of the queue.